src/rgw/const.py

```diff
@@ -83,12 +83,14 @@
                       MOTR_MAX_RPC_MSG_SIZE_KEY, MOTR_RECONNECT_INTERVAL_KEY,
                       MOTR_RECONNECT_RETRY_COUNT_KEY]
 
+
```

Comment:
remove extra line

Reply:
![image](https://user-images.githubusercontent.com/66383213/158815854-546c87ee-0fc3-48b8-8611-6be7c1cb1708.png)

There should be two blank lines between class definitions(as per python pep-8 coding standard)

Reply:
i realized later